docs/dev/design/sphinx-jquery.rst

```diff
@@ -68,24 +68,18 @@ If you are depending on a theme or extension that did not itself address the rem
 #. Add ``sphinxcontrib.jquery`` to your ``extensions`` setting in ``conf.py``.
 
 
-``sphinxcontrib.jquery.add_jquery(app)``
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``sphinxcontrib.jquery.add_jquery(app, force=False)``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Adds jQuery no matter what, but at most once.
-It's useful for themes and extensions that want jQuery added unconditionally or if they want to handle the conditions themselves.
-
-.. warning::
-
-  If you call this function, you most likely should also add ``Sphinx>=6`` to your theme's/extension's dependencies since versions before this already bundles jQuery!
+Adds jQuery if Sphinx is from version 6 and up. Adds jQuery at most once.
 
+When setting ``force=True``, adds jQuery no matter what, but at most once. This is useful if you want to handle alternative conditions for adding jQuery.
 
-``sphinxcontrib.jquery.add_jquery_if_sphinx_gte_6(app)``
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Adding ``sphinxcontrib.jquery`` to a documentation project's ``conf.extensions`` will call this function with ``force=False``.
 
-Adds jQuery if Sphinx is from version 6 and up. Adds jQuery at most once.
-This is recommended for extensions and themes that maintain compatibility of Sphinx before and after version 6.
+.. warning::
 
-Adding ``sphinxcontrib.jquery`` to a documentation project's ``conf.extensions`` will call this function.
+  If you call this function with ``force=False``, you most likely should also add ``Sphinx>=6`` to your theme's/extension's dependencies since versions before this already bundles jQuery!
```

Comment:
I think this is not what we want. The idea of the `force=False` argument was to do not have this condition.

* Using `force=False` will only add jQuery if Sphinx>=6
* When `force=True` will add jQuery _always_, not matter what

That way, themes calling this function with `force=False` and using Sphinx>6 will get the jQuery added. Also, themes using Sphinx<6 won't get jQuery added.

Reply:
Totally agree, it should say:

```
If you call this function with ``force=True``
```